countdown_timer.jsx

```diff
@@ -126,7 +126,7 @@ var CountdownTimer = React.createClass({
     var timeRemaining = this.state.timeRemaining;
 
     return (
-      <div className='timer'>
+      <div className={this.props.setClass || 'timer'}>
```

Comment:
Probably worth always including the `timer` class as well as the prop classname.

Something like `var className = 'timer ' +  (this.props.className || '')`